<?xml version="1.0"?>
<launch>
    <node pkg="cbf_pc_selector" type="cbf_pc_selector_node" name="cbf_pc_selector" output="screen">
        <rosparam file="$(find cbf_pc_selector)/config/kookaburra.yaml" command="load"/>
        <remap from="/cbf_pc_selector/camera_info" to="/d455/depth/camera_info"/>
        <remap from="/cbf_pc_selector/input_image" to="/d455/depth/image_rect_raw"/>
        <remap from="/cbf_pc_selector/output_mavros" to="/mavros/obstacles"/>
    </node>
</launch>
